build-scripts/problem-matchers/catch2.json

```
@@ -4,19 +4,19 @@
             "owner": "cata-catch2",
             "pattern": [
                 {
-                    "regexp": "^(?:.\\[[0-9;]+m)*(?:\\.\\./)*([^:]*):(\\d+): .*FAILED:.*$",
+                    "regexp": "^(?:\\[.*\\]=>)(?:.\\[[0-9;]+m)*(?:\\.\\./)*([^:]*):(\\d+): .*FAILED:.*$",
```

```suggestion
                    "regexp": "^(?:\\[.*\\]=>[ \t]*)?(?:.\\[[0-9;]+m)*(?:\\.\\./)*([^:]*):(\\d+): .*FAILED:.*$",
```
Or `\s*` instead of `[ \t]*` if this variety of regular expressions allows for it - same thing below.